src/Microsoft.Extensions.Localization/ResourceManagerStringLocalizerFactory.cs

```diff
@@ -50,6 +50,29 @@ public class ResourceManagerStringLocalizerFactory : IStringLocalizerFactory
             }
         }
 
+        protected virtual string GetKey(TypeInfo typeInfo)
+        {
+            return GetKey(typeInfo, _hostingEnvironment.ApplicationName, _resourcesRelativePath);
+        }
+
+        protected virtual string GetKey(TypeInfo typeInfo, string location, string resourcesRelativePath)
+        {
+            return string.IsNullOrEmpty(resourcesRelativePath)
```

Comment:
These need docs as users can override this functionality (instead of the comment i was asking for you can probably add a remark to the doc comment itself)